api/http/api_inventory.go

```diff
@@ -899,6 +901,42 @@ func (i *inventoryHandlers) InternalDevicesStatusHandler(w rest.ResponseWriter,
 	w.WriteJson(result)
 }
 
+func (i *inventoryHandlers) GetDeviceGroupInternalHandler(w rest.ResponseWriter, r *rest.Request) {
+	ctx := r.Context()
+
+	l := log.FromContext(ctx)
+
+	tenantId := r.PathParam("tenant_id")
+	if tenantId == "" {
```

Comment:
I know this end-point is needed by enterprise packages, but is it really needed to make this end-point multi-tenant only? In other places in the code, we accept an empty tenantID.

Reply:
fixed; this was mistake done when backporting from enterprise